src/Client/Web/Bit.Client.Web.BlazorUI/Components/SpinButton/BitSpinButton.razor.cs

```diff
@@ -17,9 +18,14 @@ public partial class BitSpinButton
         private int precision;
         private double min;
         private double max;
+        private string? intermediateValue;
         private string InputId = $"input{Guid.NewGuid()}";
-        private string IntermediateValue = string.Empty;
         private Timer? timer;
+        private ElementReference inputRef;
+        private ElementReference buttonIncrement;
+        private ElementReference buttonDecrement;
+
+        [Inject] public IJSRuntime? JSRuntime { get; set; }
```

Comment:
plz remove the nullable `?` and use `default!` as an initial value here, and also fix all `JSRuntime!` by removing the `!` character.